pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[build-system]
+# Defined by PEP 518
+requires = [
+  "setuptools>=42",
+  "setuptools_scm[toml]>=6.0",
+  "wheel",
+]
+# Defined by PEP 517
+build-backend = "setuptools.build_meta"
+
+[tool.black]
+line-length = 79
+target-version = ['py37', 'py38', 'py39']
+include = '\.pyi?$'
+exclude = '''
+
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | compiled_krb
+    | dist
+    | etc
+    | iris_image_test_output
+    | sphinxext
+    | tools
+  )/
+  | gitwash_dumper.py
+  | _ff_cross_references.py  # auto-generated files
+  | std_names.py
+  | um_cf_map.py
```

Comment:
These three should probably be removed?

Reply:
Done - and dumped `gitwash_dumper.py` as that's a hangover from `iris` docs building and has no place here :+1: